BioForm/profiles/admin.py

```diff
@@ -1,6 +1,17 @@
 from django.contrib import admin
-
-from . models import Bio
+from django.contrib.auth.admin import UserAdmin
+from . models import Bio, Account
 
 # Register your models here.
+
+class AccountAdmin(UserAdmin):
+    list_display = ('email', 'username', 'date_joined', 'last_login', 'is_admin', 'is_staff')
+    search_fields = ('email', 'username')
+    readonly_fields = ('id', 'date_joined', 'last_login')
+
+    filter_horizontal = ()
+    list_filter = ()
+    fieldsets = ()
```

Comment:
Do we need to declare these fields manually with empty tuples?

Reply:
Removing these gave the following error:
<img width="1145" alt="image" src="https://user-images.githubusercontent.com/110596170/188614170-d2f659d9-081b-462c-a72a-8f1f5919a4db.png">

This is telling us that Django expects us to have filtering options but since my focus was not on filtering options, I decided to set them as none and continue with the task at hand for now.